packages/devtools-plugin/src/devtools.plugin.ts

```diff
@@ -39,7 +39,8 @@ export class NgxsReduxDevtoolsPlugin implements NgxsPlugin {
           this.devtoolsExtension.init(state);
         } else {
           const type = getActionTypeFromInstance(action);
-          this.devtoolsExtension.send({ ...action, type }, newState);
+
+          this.devtoolsExtension.send({ ...action, type: type.desc || type }, newState);
```

Comment:
Can we make a helper method that will just get this for us?

Reply:
maybe getActionTypeFromInstance could return either or a string or and ActionToken.

Reply:
IDK, wouldn't u still want the raw type sometimes? Making that return that could be undesirable at some times.

Reply:
I added a method that returns just the name and another that returns the string or the ActionToken